8/README.md

```diff
@@ -0,0 +1,164 @@
+---
+domain: rfc.tango-controls.org
+shortname: 8/SERVER
+name: Device Server Model
+status: raw
+editor: 
+contributors: Piotr Goryl, Vincent Hardion
+---
+
+This document provides the specification of the Device Server.
+
+See also: 2/Device
+
+
+## Preamble
+
+Copyright (c) 2019 Tango Community.
+
+This Specification is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version. This Specification is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program; if not, see <http://www.gnu.org/licenses>.
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+
+## Tango Device Server specification
+
+ This spefication formally document the Tango Device Server model. 
+ Runtime implementation of the Device Server model in conforming Tango implementations MUST follow this specification.
+
+### Goals
+
+ In a Tango Control System a Device Server is in charge of managing the life cycle and the communication protocol of one or several Tango Devices.
+ The Device Server can also offer different services to the clients as well as for the Device.
+
+Additionally, it aims to:
+
+* Provide a blackbox that registers the operations executed on every Device
+
+* Be usable as a Device
+
+### Use Cases
+
+The main use cases for Device Server are:
+
+* to manage several Device in the same process. 
+
+* to share the same channel of communication
+
+* to provide the connection protocols for accessing the managed Device
+
+* to provide different services like logging
+
+* to serialise the operation per Device
```

Comment:
```suggestion
* to serialize the operations per Device
```

Reply:
s -> british
z -> us
I see your preference ;-)

Reply:
oops, sorry. You can keep the british version ;)